### PR TITLE
MCR-2054 process current solr configuration

### PR DESCRIPTION
**Pull request by rochpaul**
Recognize current localized solr components and update them

[Link to jira](https://mycore.atlassian.net/browse/MCR-2054).

### CODE REVIEW

File: mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java [hunk 1]
```diff
@@ -28,23 +28,32 @@
...
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
 import org.apache.http.client.methods.HttpPost;
 import org.apache.http.entity.StringEntity;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.mycore.common.MCRException;
```
Comment (adlerre) [inline mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java:44]: Codacy Issue found: [Avoid unused imports such as 'org.mycore.common.MCRException'](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5581923)

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5581923&bid=18296627)

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5581923&bid=18296627)

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 3075218ffc480fd717bb68054328bcffc2195810! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5591-pull_request-974

**Review by kkrebs — APPROVED** [review]
successfully tested with MIR

**Pull request merged**